lib/matplotlib/dates.py

```diff
@@ -82,11 +82,12 @@
      In [1]: date(2006, 4, 1).toordinal() - date(1, 1, 1).toordinal()
      Out[1]: 732401
 
-All the Matplotlib date converters, tickers and formatters are timezone aware.
-If no explicit timezone is provided, :rc:`timezone` is assumed.  If you want to
-use a custom time zone, pass a `datetime.tzinfo` instance with the tz keyword
-argument to `num2date`, `.Axis.axis_date`, and any custom date tickers or
-locators you create.
+All the Matplotlib date converters, tickers and formatters are time zone aware.
```

Comment:
python uses "timezone", and we usually follow US English, which also favours "timezone":

```suggestion
All the Matplotlib date converters, tickers and formatters are timezone aware.
```

Reply:
... and the changes below...

Reply:
But I am quite convinced that it should be spelled time zone otherwise. https://en.wikipedia.org/wiki/Time_zone

Reply:
Ah, sorry, I didn't see the comment above the change. I'll change everything to timezone then. (Even the old time zone)

Reply:
Thats OK, everyone gets me on "co-ordinate"....